test/c/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+# This file is inspired by the description of handling a larger number of tests
+# provided here:
+# https://cmake.org/cmake/help/book/mastering-cmake/chapter/Testing%20With%20CMake%20and%20CTest.html
+cmake_minimum_required (VERSION 3.27)
```

Comment:
Just curious how you picked 3.27 here? It's not a big deal since this is just for testing.

Reply:
@Cryoris can you comment on this? Looks like you added that here: https://github.com/Cryoris/qiskit-terra/commit/029018843b01a50ab92a0cf4d384105827e780be

Reply:
Oh that was just the version I happened to have installed locally -- we can also change this to something lower, not sure at which point we start losing features we need 🙂

Reply:
It doesn't necessarily need to be lower. I was just curious how you settled on that. I'd say we should be conservative if this was needed for building more broadly, but since it's only for testing it should be fine as is.